datadog_checks_base/datadog_checks/base/utils/db/core.py

```diff
@@ -40,10 +40,10 @@ def __init__(self, check, executor, queries=None, tags=None, error_handler=None)
     def compile_queries(self):
```

Comment:
Could you split into 2 PR?
- The signature change
- The service checks feature

Easier to review, and better commit history.